static/css/jira-connect-ghe.css

```diff
@@ -0,0 +1,55 @@
+.jiraConnectGHE__content {
+  margin: 4em auto 0;
+  max-width: 560px;
+}
+.jiraConnectGHE__content__title {
+  text-align: center;
+  margin-bottom: 32px;
+}
+
+.jiraConnectGHE__options__subTitle {
+  font-size: 16px;
+  margin-bottom: 8px;
+  display: inline-flex;
+}
+
+.jiraConnectGHE__prompt {
+  color: #344563;
+  font-size: 14px;
+  margin-bottom: 16px;
+}
+
+.jiraConnectGHE__error {
+  display: none;
+  color: #DE350B;
+  font-size: 12px;
+  margin-top: 2px;
+}
+
+label {
+  font-size: 12px;
+  font-weight: 600;
+  color: #6B778C;
+}
+
+.ghe-server-input {
```

Comment:
Why has this class name broken from the jiraConnectGH__ pattern? All class names in this file should be prefixed with the same thing.